src/app/shared/form/builder/ds-dynamic-form-ui/models/onebox/dynamic-onebox.component.ts

```diff
@@ -0,0 +1,276 @@
+import { ChangeDetectorRef, Component, EventEmitter, Input, OnInit, Output, ViewChild } from '@angular/core';
+import { FormGroup } from '@angular/forms';
+
+import { DynamicFormLayoutService, DynamicFormValidationService } from '@ng-dynamic-forms/core';
+import {
+  catchError,
+  debounceTime,
+  distinctUntilChanged,
+  filter,
+  map,
+  merge,
+  switchMap,
+  take,
+  tap
+} from 'rxjs/operators';
+import { Observable, of as observableOf, Subject, Subscription } from 'rxjs';
+import { NgbModal, NgbModalRef, NgbTypeahead, NgbTypeaheadSelectItemEvent } from '@ng-bootstrap/ng-bootstrap';
+
+import { VocabularyService } from '../../../../../../core/submission/vocabularies/vocabulary.service';
+import { DynamicOneboxModel } from './dynamic-onebox.model';
+import { hasValue, isEmpty, isNotEmpty, isNotNull } from '../../../../../empty.util';
+import { FormFieldMetadataValueObject } from '../../../models/form-field-metadata-value.model';
+import { ConfidenceType } from '../../../../../../core/shared/confidence-type';
+import { getFirstSucceededRemoteDataPayload } from '../../../../../../core/shared/operators';
+import { PaginatedList } from '../../../../../../core/data/paginated-list';
+import { VocabularyEntry } from '../../../../../../core/submission/vocabularies/models/vocabulary-entry.model';
+import { PageInfo } from '../../../../../../core/shared/page-info.model';
+import { DsDynamicVocabularyComponent } from '../dynamic-vocabulary.component';
+import { Vocabulary } from '../../../../../../core/submission/vocabularies/models/vocabulary.model';
+import { VocabularyTreeviewComponent } from '../../../../../vocabulary-treeview/vocabulary-treeview.component';
+import { VocabularyEntryDetail } from '../../../../../../core/submission/vocabularies/models/vocabulary-entry-detail.model';
+
+@Component({
+  selector: 'ds-dynamic-onebox',
+  styleUrls: ['./dynamic-onebox.component.scss'],
+  templateUrl: './dynamic-onebox.component.html'
+})
+export class DsDynamicOneboxComponent extends DsDynamicVocabularyComponent implements OnInit {
```

Comment:
Please add TypeDocs at the class level to describe this new component.

Reply:
done